config/waagent.logrotate

```diff
@@ -1,7 +1,22 @@
 /var/log/waagent.log {
+    # Old versions of log files are compressed with gzip by default.
     compress
-    monthly
-    rotate 6
+
+    # Rotate log files > 20 MB, and keep last 50 archived files. With a compression ratio of around 10:1, The archived
```

Comment:
how did you get this ratio? thanks

Reply:
This number is anecdotal, from trying different waagent.log files.

```
v@dev:/var/log$ ls -l waagent.log*
-rw-r--r-- 1 root root  9461 Dec 26 17:17 waagent.log
-rw-r--r-- 1 root root  5968 Dec  1 04:51 waagent.log.1.gz
-rw-r--r-- 1 root root 26143 Nov  1 05:39 waagent.log.2.gz
v@dev:/var/log$ gunzip -c waagent.log.1.gz | wc --bytes
57636
v@dev:/var/log$ gunzip -c waagent.log.2.gz | wc --bytes
285857
```